docs/design/coreclr/profiling/davbr-blog-archive/Profiler stack walking Basics and beyond.md

```diff
@@ -17,31 +17,31 @@ It’s nice to be able to get call stacks whenever you want them.  But with powe
 
 So let’s take a look at the beast.  Here’s what your profiler calls (you can find this in ICorProfilerInfo2, in corprof.idl):
 ```
-HRESULT DoStackSnapshot(   
-  [in] ThreadID thread,   
-  [in] StackSnapshotCallback *callback,   
-  [in] ULONG32 infoFlags,   
-  [in] void *clientData,   
-  [in, size_is(contextSize), length_is(contextSize)] BYTE context[],   
+HRESULT DoStackSnapshot(
+  [in] ThreadID thread,
+  [in] StackSnapshotCallback *callback,
+  [in] ULONG32 infoFlags,
+  [in] void *clientData,
+  [in, size_is(contextSize), length_is(contextSize)] BYTE context[],
   [in] ULONG32 contextSize);
 ```
 And here’s what the CLR calls on your profiler (you can also find this in corprof.idl).  You’ll pass a pointer to your implementation of this function in the callback parameter above.
 ```
-typedef HRESULT \_\_stdcall StackSnapshotCallback(   
-  FunctionID funcId,   
-  UINT_PTR ip,   
-  COR_PRF_FRAME_INFO frameInfo,   
-  ULONG32 contextSize,   
-  BYTE context[],   
+typedef HRESULT \_\_stdcall StackSnapshotCallback(
```

Comment:
@jkotas I believe the backslashes here are extra. Underscores shouldn't be escaped in a codeblock. Can you confirm?

```suggestion
typedef HRESULT __stdcall StackSnapshotCallback(
```